cpg-concepts/src/integrationTest/kotlin/de/fraunhofer/aisec/cpg/concepts/file/FileConceptTest.kt

```diff
@@ -62,7 +62,10 @@ class FileConceptTest : BaseTest() {
         assertTrue(fileNodes.isNotEmpty())
```

Comment:
Did you try to re-enable the ignored test?

Reply:
yes and it seems to work